### PR TITLE
Got custom screen and logo to work and updated lang file

### PR DESCRIPTION
**Pull request by Nieadni**
[Got custom screen and logo to work and updated lang file](https://github.com/ManasMods/reincarnated_mod/commit/585beac1bb6e5fc6600eafda8d7ea5f853f5df2a)

### CODE REVIEW

File: src/main/java/com/github/manasmods/tensura/event/TensuraEvents.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import net.minecraftforge.fml.common.Mod;
+
+@Mod.EventBusSubscriber(modid = Tensura.MOD_ID)
+public class TensuraEvents {
+
+    @SubscribeEvent
+    @OnlyIn(Dist.CLIENT)
+    public static void openGui(ScreenOpenEvent event) {
+        if (event.getScreen() instanceof TitleScreen && !(event.getScreen() instanceof TensuraTitleScreen)) {
+            event.setScreen(new TensuraTitleScreen());
```
Comment (Charismara) [inline src/main/java/com/github/manasmods/tensura/event/TensuraEvents.java:19]: what are u trying to achive? Why are u replacing the whole title screen

**Review by Charismara — CHANGES_REQUESTED** [review]
last review for the next week

File: src/main/java/com/github/manasmods/tensura/event/TensuraEvents.java (hunk 1 above)
Comment (Nieadni) [reply src/main/java/com/github/manasmods/tensura/event/TensuraEvents.java:19]: More proof of concept. We may aswell have a custom title screen for release so I was just testing it

File: src/main/java/com/github/manasmods/tensura/event/TensuraEvents.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        }
+    }
```
[the remark this answers is not in the document]
Comment (Nieadni) [reply src/main/java/com/github/manasmods/tensura/event/TensuraEvents.java:21]: done

**Comment by Nieadni** [comment]
> last review for the next week

Bruh. Its only tuesday

**Comment by Nieadni** [comment]
┬┴┬┴┤(･_├┬┴┬┴

File: src/main/java/com/github/manasmods/tensura/client/TensuraClient.java [hunk 3]
```diff
@@ -32,4 +37,12 @@ private static void registerEntityRenderer() {
         EntityRenderers.register(TensuraEntityTypes.GIANT_COD.get(), GiantCodRenderer::new);
         EntityRenderers.register(TensuraEntityTypes.ONE_EYED_OWL.get(), OneEyedOwlRenderer::new);
     }
+
+    public TensuraClient() {MinecraftForge.EVENT_BUS.addListener(this::openGui);}
+    public void openGui(ScreenOpenEvent event) {
+        if (event.getScreen() instanceof TitleScreen && !(event.getScreen() instanceof TensuraTitleScreen)) {
+            event.setScreen(new TensuraTitleScreen());
+        }
+    }
```
Comment (Charismara) [inline src/main/java/com/github/manasmods/tensura/client/TensuraClient.java:46]: Add the listener in `Tensura` using a DistExecutor

**Review by Charismara — CHANGES_REQUESTED** [review]

File: src/main/java/com/github/manasmods/tensura/Tensura.java [hunk 4]
```diff
@@ -39,6 +34,7 @@ public Tensura() {
         modEventBus.addListener(AttributeHandler::onAttributeModification);
         modEventBus.addListener(CapabilityHandler::registerCapabilities);
         modEventBus.addListener(TensuraEntityHandler::entityAttributeEvent);
+        forgeEventBus.addListener(TensuraClient::openGui);
```
Comment (Charismara) [inline src/main/java/com/github/manasmods/tensura/Tensura.java:37]: Please use a DistExecutor to only add that listener on client side

**Review by Charismara — CHANGES_REQUESTED** [review]

**Review by Charismara — APPROVED** [review]

**Pull request merged**